builtin/logical/pki/path_tidy.go

```diff
@@ -657,6 +679,13 @@ available on the tidy-status endpoint.`,
 
 func (b *backend) pathTidyWrite(ctx context.Context, req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
 	safetyBuffer := d.Get("safety_buffer").(int)
+	revokedSafetyBuffer := 0
+	if revokedSafetyBufferRaw, ok := d.GetOk("revoked_safety_buffer"); ok {
+		revokedSafetyBuffer = revokedSafetyBufferRaw.(int)
+	} else {
+		// Default to safety buffer's default if missing
+		revokedSafetyBuffer = int(defaultTidyConfig.SafetyBuffer.Seconds())
```

Comment:
I think we just want to default to the safety buffer's value, not to its default here. Otherwise it is a behavior change.

Reply:
Alright, I don't remember why I did that... this is better :D